tests/web/test_client.py

```diff
@@ -1112,6 +1112,19 @@ def testVersion6GetWithLessThanEqualToMemory(self):
         <patch type="complete" URL="http://a.com/f" hashFunction="sha512" hashValue="34" size="33"/>
     </update>
 </updates>
+""",
+        )
+
+    def testVersion6GetWithLessThanEqualToMemoryEncoded(self):
+        ret = self.client.get("/update/6/f/1.0/1/p/l/a/a/ISET%3aSSE3,MEM%3a8000/a/a/update.xml")
```

Comment:
Hmm I need to quote commas as well not just colons.  Will update the patch on Monday.